hrm/department_app/tests/test_department_api.py

```diff
@@ -0,0 +1,64 @@
+from django.urls import reverse
+from rest_framework import status
+from django.test import Client
+from rest_framework.test import APITestCase
+from department_app.models.department_models import Department
+from department_app.views.department_views import DepartmentViewSet
+import json
+
+
+class DepartmentTests(APITestCase):
+    def test_post_department(self):
+        url = reverse('department_app:departments-list')
+        data = {'name': 'Develope', 'status': 'Active'}
+        response = self.client.post(url, data, format='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(Department.objects.get(
+            pk=response.json()['id']).name, 'Develope')
+        self.assertEqual(Department.objects.get(
+            pk=response.json()['id']).status, 'Active')
+
+    def test_get_department(self):
+        url = reverse('department_app:departments-list')
+        response = self.client.get(url, format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
Here you should check not only the response status but also the data returned by your request.

Reply:
I already fixed it.